Platform/OpenCanopy/Views/BootPicker.c

```diff
@@ -418,8 +418,8 @@ InternalBootPickerEntryDraw (
   //
   // Draw the label horizontally centered.
   //
-  //ASSERT (Label->Width  <= BOOT_ENTRY_DIMENSION);
-  //ASSERT (Label->Height == BOOT_ENTRY_LABEL_HEIGHT);
+  //ASSERT (Label->Width  <= BOOT_ENTRY_DIMENSION * DrawContext->Scale);
+  ASSERT_EQUALS (Label->Height, BOOT_ENTRY_LABEL_HEIGHT * DrawContext->Scale);
```

Comment:
I think this wants to become <= following the `APPLE_DISK_LABEL_MAX_WIDTH` check during loading.

Also, can we have static assertions for these dimensions being <= `APPLE_DISK_LABEL_MAX_WIDTH` and `APPLE_DISK_LABEL_MAX_HEIGHT`.

Reply:
<= or >=? 

We can also test (on Mac) if Apple enforces these requirements, e. g. what if we generate a  64x64 label.

Reply:
That height does not exceed 12. Testing on a Mac is a good idea, though I would say we should be better than them in terms of restrictions.